components/browser/state/src/test/java/mozilla/components/browser/state/action/WebExtensionActionTest.kt

```diff
@@ -249,10 +251,25 @@ class WebExtensionActionTest {
         store.dispatch(WebExtensionAction.InstallWebExtensionAction(extension)).joinBlocking()
 
         assertEquals(extension, store.state.extensions[extension.id])
-        assertNull(store.state.extensions[extension.id]?.browserActionPopupSession)
+        assertNull(store.state.extensions[extension.id]?.popupSessionId)
 
         store.dispatch(WebExtensionAction.UpdatePopupSessionAction(extension.id, "popupId")).joinBlocking()
-        assertEquals("popupId", store.state.extensions[extension.id]?.browserActionPopupSession)
+        assertEquals("popupId", store.state.extensions[extension.id]?.popupSessionId)
+    }
+
+    @Test
+    fun `UpdateActionPopupSession - Adds popup engine session to the web extension state`() {
```

Comment:
nit: This is a typo (also in the existing test above, should be: `UpdatePopupSessionAction`).